source/PreferencesPanel.cpp

```diff
@@ -710,10 +714,20 @@ void PreferencesPanel::DrawSettings()
 		}
 		else if(setting == EXPEND_AMMO)
 			text = Preferences::AmmoUsage();
+		else if(setting == DATE_FORMAT)
+		{
+			isOn = true;
+			Preferences::DateFormat datefmtShortName = Preferences::GetDateFormat();
+			if(datefmtShortName == Preferences::DateFormat::ymd)
+				text = "yyyy-mm-dd";
+			else if(datefmtShortName == Preferences::DateFormat::dmy)
+				text = "dd/mm/yyyy";
+			else if(datefmtShortName == Preferences::DateFormat::mdy)
+				text = "mm/dd/yyyy";
+		}
 		else if(setting == FLOTSAM_SETTING)
 		{
 			text = Preferences::FlotsamSetting();
```

Comment:
Why is this changed?
```suggestion
			text = Preferences::FlotsamSetting();
			isOn = text != "off";
```